api4/cloud.go

```diff
@@ -411,6 +412,22 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = appErr
 			return
 		}
+	case model.EventTypeTrialWillEnd:
+		user, appErr := c.App.GetUserByEmail(event.CloudWorkspaceOwner.UserEmail)
+		if appErr != nil {
+			c.Err = model.NewAppError("Api4.handleCWSWebhook", appErr.Id, nil, appErr.Error(), appErr.StatusCode)
+			return
+		}
+
+		endTimeStamp := event.SubscriptionTrialEndUnixTimeStamp
+		t := time.Unix(endTimeStamp, 0)
+		trialEndDate := fmt.Sprintf("%s %d, %d", t.Month(), t.Day(), t.Year())
```

Comment:
While this will probably work 99.9% of the time, I think you're better off using [time.Format](https://golang.org/pkg/time/#Time.Format)

Reply:
I couldn't quite get the right layout that looked similar to this